{"errorMessage":"Lambda process returned error status code: 1. Result: {\"errorType\":\"NetworkingError\",\"errorMessage\":\"connect ECONNREFUSED 127.0.0.1:4566\"}. Output:\n\u001b[32mSTART RequestId: 7ac3dd20-3021-1a99-d2d8-3e0adbd84d16 Version: $LATEST\u001b[0m\n2021-08-05T11:28:30.415Z\t7ac3dd20-3021-1a99-d2d8-3e0adbd84d16\tINFO\tERR Error: connect ECONNREFUSED 127.0.0.1:4566\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16) {\n  errno: 'ECONNREFUSED',\n  code: 'NetworkingError',\n  syscall: 'connect',\n  address: '127.0.0.1',\n  port: 4566,\n  region: 'ap-southeast-2',\n  hostname: 'localhost',\n  retryable: true,\n  time: 2021-08-05T11:28:30.406Z\n}\n2021-08-05T11:28:30.420Z\t7ac3dd20-3021-1a99-d2d8-3e0adbd84d16\tINFO\tnull\n2021-08-05T11:28:30.426Z\t7ac3dd20-3021-1a99-d2d8-3e0adbd84d16\tERROR\tInvoke Error \t{\"errorType\":\"NetworkingError\",\"errorMessage\":\"connect ECONNREFUSED 127.0.0.1:4566\",\"code\":\"NetworkingError\",\"message\":\"connect ECONNREFUSED 127.0.0.1:4566\",\"errno\":\"ECONNREFUSED\",\"syscall\":\"connect\",\"address\":\"127.0.0.1\",\"port\":4566,\"region\":\"ap-southeast-2\",\"hostname\":\"localhost\",\"retryable\":true,\"time\":\"2021-08-05T11:28:30.406Z\",\"stack\":[\"Error: connect ECONNREFUSED 127.0.0.1:4566\",\"    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16)\"]}\n\u001b[32mEND RequestId: 7ac3dd20-3021-1a99-d2d8-3e0adbd84d16\u001b[0m\n\u001b[32mREPORT RequestId: 7ac3dd20-3021-1a99-d2d8-3e0adbd84d16\tInit Duration: 3000.00 ms\tDuration: 381.07 ms\tBilled Duration: 382 ms\tMemory Size: 1536 MB\tMax Memory Used: 58 MB\t\u001b[0m","errorType":"InvocationException","stackTrace":["  File \"/opt/code/localstack/localstack/services/awslambda/lambda_api.py\", line 739, in run_lambda\n    callback=callback,\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 224, in execute\n    return do_execute()\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 214, in do_execute\n    return _run(func_arn=func_arn)\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 157, in wrapped\n    raise e\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 153, in wrapped\n    result = func(*args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 201, in _run\n    raise e\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 189, in _run\n    result = self._execute(func_arn, func_details, event, context, version)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 535, in _execute\n    return super(LambdaExecutorReuseContainers, self)._execute(func_arn, *args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 451, in _execute\n    cmd, event=stdin, env_vars=environment, func_details=func_details\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 342, in run_lambda_executor\n    result,\n"]}
